README.md

```diff
@@ -1,34 +1,67 @@
-
-Documentation: https://docs.viam.com/components/base/agilex-limo/
+This module implements the base driver for the [AgileX LIMO](https://global.agilex.ai/products/limo) base to be used with [`viam-server`](https://docs.viam.com/). This driver supports differential, ackermann, and omni directional steering modes over the serial port.
 
 ## Build and Run
 
-The module is avaiable for configuration and use via the Viam Registry at https://app.viam.com/module/viam/agilex-limo.
+To use this module, follow these instructions to [add a module from the Viam Registry](https://docs.viam.com/modular-resources/configure/#add-a-module-from-the-viam-registry) and select the `viam:base:agilex-limo` model from the [`agilex-limo` module](https://app.viam.com/module/viam/agilex-limo).
 
-For local development, this module is written in Go.
+## Configure your AgileX LIMO base
 
-To build: `make limobase`<br>
-To test: `make test`
+> [!NOTE]  
+> Before configuring your base, you must [create a robot](https://docs.viam.com/manage/fleet/robots/#add-a-new-robot).
 
-## Configuration
+Navigate to the **Config** tab of your robot’s page in [the Viam app](https://app.viam.com/). Click on the **Components** subtab and click **Create component**. Select the `base` type, then select the `agilex-limo` model. Enter a name for your base and click **Create**.
 
-```javascript
+On the new component panel, copy and paste the following attribute template into your base’s **Attributes** box. 
+```json
 {
-  "drive_mode": "", //Required. valid values are "differential", "ackermann", or "omni"
-  "serial_path": "" //Optional. serial path for communication. Defaults to "/dev/ttyTHS1"
+  "drive_mode": "<ackermann|differential|omni>",
+  "serial_path": "<your-serial-path>"
 }
 ```
 
-Example:
+> [!NOTE]  
+> For more information, see [Configure a Robot](https://docs.viam.com/manage/configuration/).
+
+### Attributes
+
+The following attributes are available for `viam:base:agilex-limo` bases:
 
-```javascript
+| Name | Type | Inclusion | Description |
+| ---- | ---- | --------- | ----------- |
+| `drive_mode` | string | **Required** | LIMO [steering mode](https://docs.trossenrobotics.com/agilex_limo_docs/operation/steering_modes.html#switching-steering-modes). Options: `differential`, `ackermann`, `omni` (mecanum). |
+| `serial_path` | string | **Required** | The full filesystem path to the serial device, starting with <file>/dev/</file>. With your serial device connected, you can run `sudo dmesg \| grep tty` to show relevant device connection log messages, and then match the returned device name, such as `ttyTHS1`, to its device file, such as <file>/dev/ttyTHS1</file>. If you omit this attribute, Viam will attempt to automatically detect the path.<br>Default: `/dev/ttyTHS1` |
```

Comment:
same. Optional. Also "If you omit this attribute, Viam will attempt to automatically detect the path." isn't right, we'll just default to a value without "auto detecting".